Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:16.04
+MAINTAINER nop@nop.com
+
+RUN apt-get update -q
+RUN apt-get install --yes build-essential
```

Comment:
You use `-y` elsewhere and `--yes` here. Could you update for consistency?

Reply:
Done.